nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -128,6 +128,16 @@ in
         '';
       };
 
+      firewallOpenPorts = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Specifies whether the ports the SSH daemon listens on must
+          be enabled in the firewall. Remember to open at least some
+          of them.
```

Comment:
That's a great change (imho) - related to #19504 

> For example, enabling a module should not open firewall ports by default.

I don't think we should immediately use `default = false` but we should make it clear in the description that this will probably happen in the future and that one shouldn't enable this option (i.e. use the firewall module instead).

Additional information can be found in the related issue: #19504